generator/routing_index_generator.cpp

```diff
@@ -172,10 +173,42 @@ bool RegionsContain(vector<m2::RegionD> const & regions, m2::PointD const & poin
   return false;
 }
 
+double CalcClockwiseDistance(vector<m2::RegionD> const & borders,
+                             CrossMwmConnectorSerializer::Transition const & transition)
+{
+  double distance = 0.0;
+
+  for (m2::RegionD const & region : borders)
+  {
+    vector<m2::PointD> points = region.Data();
+
+    m2::PointD const * prev = &points.back();
+    for (m2::PointD const & curr : points)
+    {
+      m2::PointD intersection;
+      if (region.IsIntersect(transition.GetBackPoint(), transition.GetFrontPoint(), *prev, curr,
```

Comment:
Is it possible to mark Region2D::IsIntersect as static? If so, please, do that. Otherwise I don't see how the contents of region is used in IsIntersect().